ci-operator/step-registry/ibmcloud/provision/dedicatedhost/static/ibmcloud-provision-dedicatedhost-static-commands.sh

```diff
@@ -0,0 +1,155 @@
+    family=${tmp%,*}
+    class=${tmp#*,}
+    echo "family:" $family " class:" $class
+    dhg="${dhName}g"
+    run_command "${IBMCLOUD_CLI} is dedicated-host-group-create --zone $zone --family $family --class $class --name ${dhg}"
+    
+    ${IBMCLOUD_CLI} is dedicated-host-create --profile $dProfile --name $dhName --dhg $dhg --output JSON
+    
+    status=$(waitingStatus ${dhName})
+    echo "${dhName} status: ${status}"
+    run_command "${IBMCLOUD_CLI} is dedicated-host ${dhName}"
+
+    if [[ "${status}" != "stable" ]]; then
+        echo "ERROR: fail to create the dedicated host with profile $dProfile on zone: $zone resouce group:$rg"
+        echo "delete the dh ${dhName}, which status is ${status} ..."
+        run_command "${IBMCLOUD_CLI} is dhd ${dhName} -f"
+        run_command "${IBMCLOUD_CLI} is dhgd ${dhg} -f"
+    else
+        echo "${dhName}" > "${file}"        
+    fi
+}
+
+# waiting the pending dh status changed
+function waitingStatus() {
+    local dh=$1 status
+    COUNTER=0
+    while [ $COUNTER -lt 200 ]
+    do 
+        sleep 10
+        COUNTER=`expr $COUNTER + 20`
+        status=$(${IBMCLOUD_CLI} is dh $dh --output JSON | jq -r ."lifecycle_state")
+        if [[ "${status}" != "pending" ]]; then
+            break
+        fi
+    done
+    echo ${status}
+}
+
+#####################################
+##############Initialize#############
+#####################################
+
+dhProfileMaster="bx2-host-152x608"
+dhProfileWorker="bx2-host-152x608"
+masterProfile="bx2-8x32"
+workerProfile="bx2-4x16"
+
+echo "the dh profile of the master nodes: ${dhProfileMaster}; the dh profile of the worker nodes ${dhProfileWorker}"
+
+echo "$(date -u --rfc-3339=seconds) - Creating the Dedicated Host..."
+CLUSTER_NAME="${NAMESPACE}-${UNIQUE_HASH}"
+
+if [[ ${dhProfileMaster} == ${dhProfileWorker} ]]; then
+    dhName_master="${CLUSTER_NAME}-dh"
+    dhName_worker="${CLUSTER_NAME}-dh"
+else
+    dhName_master="${CLUSTER_NAME}-master-dh"
+    dhName_worker="${CLUSTER_NAME}-worker-dh"
+fi
+
+dhgRGFile=${SHARED_DIR}/ibmcloud_resource_group
+dhMaster_file=${SHARED_DIR}/dedicated_host_master
+dhWorker_file=${SHARED_DIR}/dedicated_host_worker
+
+#####################################
+#######create the dedicated host#####
+#####################################
+ibmcloud_login
+
+#create the resouce group of the dedicated host group and saved in "${SHARED_DIR}/ibmcloud_resource_group"
+rg_dhg="${CLUSTER_NAME}-dhrg"
+
+echo "create resource group ... ${rg_dhg}"
+"${IBMCLOUD_CLI}" resource group-create ${rg_dhg} || exit 1
+echo "${rg_dhg}" >  ${dhgRGFile}
+
+run_command "${IBMCLOUD_CLI} target -g $rg_dhg"
+
+zone="${region}-${DEDICATEDHOST_ZONE}"
+echo "use the zone defined with DEDICATEDHOST_ZONE: ${zone}"
+
+createDH ${rg_dhg} ${dhProfileMaster} ${zone} ${dhName_master} ${dhMaster_file}
+if [[ ${dhProfileMaster} != ${dhProfileWorker} ]]; then
```

Comment:
Suggest to directly use one dhProfile to create one DH, that is simple enough, that is it.

Reply:
The dynamic workflow uses a different dhProfile, I keep it for easily reproduce the failed dynamic job.

Reply:
pls do not mix other things together in static DH case.